python/pyspark/sql/readwriter.py

```diff
@@ -616,7 +616,7 @@ def jdbc(self, url, table, column=None, lowerBound=None, upperBound=None, numPar
 class DataFrameWriter(OptionUtils):
     """
     Interface used to write a :class:`DataFrame` to external storage systems
-    (e.g. file systems, key-value stores, etc). Use :func:`DataFrame.write`
+    (e.g. file systems, key-value stores, etc). Use :attr:`DataFrame.write`
```

Comment:
Why is this `attr`? Technically this should be `meth`. Can you double check the generated documentation is correct? You can check via `cd python/docs && make clean html`

Reply:
`DataFrame.write` is defined as a function decorated with `@property`, and as such behaves as an attribute, rather than a function. Indeed, one needs to write e.g. `df.write.saveAsTable('foo')` rather than `df.write().saveAsTable('foo')`; the latter will not work.

Changing `func` to `attr` makes Sphinx render `DataFrame.write` correctly, instead of misleadingly rendering `DataFrame.write()` in the documentation.

I've double-checked the resulting documentation, looks fine to me.

Reply:
Okay.